cmd/minishift/cmd/start.go

```diff
@@ -892,3 +897,13 @@ func startTray() error {
 	}
 	return nil
 }
+
+func updateConsoleURL(output string) string {
+	numBlock := "(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])"
+	regexPattern := numBlock + "\\." + numBlock + "\\." + numBlock + "\\." + numBlock + ":" + "8443"
```

Comment:
@praveenkumar WDYT? I think we need to be restrictive to the structure in relation to: https://github.com/openshift/origin/blob/78e96f3c3d23190fb62825f5be67f16f454289f5/pkg/oc/clusterup/up.go#L363-L368

```
	masterURL := fmt.Sprintf("https://%s:6443", c.GetPublicHostName())

	msg := fmt.Sprintf("OpenShift server started.\n\n"+
		"The server is accessible via web console at:\n"+
		"    %s\n\n", masterURL)
fmt.Fprintf(out, msg)
```

Note 1: so the line is actually `    %s`, (where %s is the IP and port)
Note 2: on master this will be `6443`

Reply:
@gbraad Do not worry about `6443` on master as we will remove this workaround once "oc cluster up" provides this and we are not sure about "oc cluster up" post 3.11.

Reply:
>  I think we need to be restrictive to the structure in relation to:

@gbraad This is anyway a temp one and we don't care about master as of now since there is no by default web-console as part of `oc cluster up`.